test/ecto/migration_test.exs

```diff
@@ -292,6 +303,20 @@ defmodule Ecto.MigrationTest do
     assert result == table(:posts)
   end
 
+  test "forward: drops a table if table exists" do
+    result = drop_if_exists table(:posts)
+    flush()
+    assert {:drop_if_exists, %Table{}} = last_command()
+    assert result == table(:posts)
+  end
+
+  test "forward: drops if exists returns parameter if parameter does not exist" do
```

Comment:
It seems this is exactly the same code as the test before, and the test is using the TestAdapter, so I think this test might not be testing what it says 🙂

Reply:
In the above test it is a test for the drop behaviour.

This is a unit test on the behaviour that returns the parameter when the parameter "does not exist"

"drops if exists" in the name of this test matches the function name, is not a behaviour
If putting `drops_if_exists/1` can get lighter

Reply:
@fertapric how do I simulate a table that does not exist?

Reply:
@joaothallis I’m not sure we should simulate that at this level of the test suite. Integration tests should cover those scenarios.

Reply:
@joaothallis what do you think about removing the test then?